Rules.md

```diff
@@ -4,8 +4,7 @@
       in a game.
     * **(Command token) round:** A single iteration in round order where all players can play a
       single command token
-    * **Round order:** Player order: Starts with the order below and shifts every turn (first player
-      becomes last player every turn):
+    * **Round order:** Player order is the order below:
```

Comment:
This was introduced after a test-play because it made the game less fun for some players. I think this should be kept.